.circleci/config.yml

```diff
@@ -117,11 +117,31 @@ workflows:
           context: quay-login
           requires:
             - deploy_staging
+  perf_lighthouse_test:
```

Comment:
created this workflow so that we can check if it passed/failed and allow/disallow merge into master